README.md

```diff
@@ -27,204 +26,9 @@ In addition you have also some useful scripts to help around testing:
-  --list                Display internal data
-  --json                Display data in json format
-  --table               Result represented in tables
-  --save SAVE           Save output to file. Only valid for --list and --json
-  --all-results         Display all test cases results. Default table view (Only valid with --table)
-  --by-host             Provides summary of test results per device (Only valid with --table)
-  --by-test             Provides summary of test results per test case (Only valid with --table)
-```
-
-Default output is a table format listing all test results, and it can be changed to a report per test case or per host
-
-### Default report
-
-```bash
-$ check-devices.py -i .personal/avd-lab.yml -c .personal/ceos-catalog.yml --table
-
-                             All tests results
-┏━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━┓
-┃ Device IP     ┃ Test Name              ┃ Test Status ┃ Message(s)       ┃
-┡━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━┩
-│ 10.73.252.11  │ verify_mlag_interfaces │ success     │                  │
-│ 10.73.252.12  │ verify_mlag_interfaces │ success     │                  │
-│ 10.73.252.102 │ verify_mlag_interfaces │ skipped     │ MLAG is disabled │
-└───────────────┴────────────────────────┴─────────────┴──────────────────┘
-```
-
-### Report per test case
-
-```
-$ check-devices.py -i .personal/avd-lab.yml -c .personal/ceos-catalog.yml --table --by-test --test verify_mlag_status
-
-                                              Summary per test case
-┏━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
-┃ Test Case          ┃ # of success ┃ # of skipped ┃ # of failure ┃ # of errors ┃ List of failed or error nodes ┃
-┡━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━╇━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┩
-│ verify_mlag_status │ 8            │ 13           │ 0            │ 0           │ []                            │
-└────────────────────┴──────────────┴──────────────┴──────────────┴─────────────┴───────────────────────────────┘
-```
-
-### Report per host
-
-```bash
-$ check-devices.py -i .personal/avd-lab.yml -c .personal/ceos-catalog.yml --table --by-host --test verify_mlag_status --hostip 10.73.252.21
-
-                                            Summary per host
-┏━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
-┃ Host IP      ┃ # of success ┃ # of skipped ┃ # of failure ┃ # of errors ┃ List of failed ortest case ┃
-┡━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━╇━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━┩
-│ 10.73.252.21 │ 0            │ 1            │ 0            │ 0           │ []                         │
-└──────────────┴──────────────┴──────────────┴──────────────┴─────────────┴────────────────────────────┘
-````
-
+# Documentation
```

Comment:
Seems everything you removed here are not pasted in `docs/README.md` so we loose a lot of initial information.